tools/backup/templates/backup.yaml

```diff
@@ -1,60 +1,77 @@
-# Sample pod to be applied manually or scheduled when desired
-# Executes a backup against the DB when tailored to the right endpoint.
-# Copies resulting full backup to $BUCKET
-apiVersion: batch/v1
-kind: Job
+apiVersion: batch/v1beta1
+kind: CronJob
 metadata:
-  name: "{{ .Release.Name }}-neo4j-backup-{{ date "20060102150405" now }}"
+  name: "{{ .Release.Name }}-job"
   labels:
     app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
     app.kubernetes.io/instance: {{ .Release.Name | quote }}
     helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     app.kubernetes.io/name: "graph-backup"
     app.kubernetes.io/component: backup
 spec:
-  backoffLimit: 3
-  template:
+  schedule: {{.Values.jobSchedule | quote }}
```

Comment:
I really like the jobSchedule approach, but please put it in an if block, and allow the user to *not* specify it for one-off backups.   I.e. automating this on a regular basis is a good idea, but between intervals, users may wish to take a one-off hot backup before performing a maintenance operation like a rolling upgrade.

Reply:
For this case, a hot backup can be created using "kubectl create job --from=cronjob/<name of cronjob> <name of job>". I can still if this situation is requested